packages/frontend-v2/pages/index.tsx

```diff
@@ -10,13 +13,332 @@ const Home: NextPage = () => {
     SeasonEnum.S2,
   ];
 
+  const [token, setToken] = useState("");
+  const [planId, setPlanId] = useState<number>();
+
+  // handler that calls fetchCourse from our api-client package
+  const testFetchCourse = (subject: string, classId: string) => {
+    SearchAPI.fetchCourse(subject.toUpperCase(), classId).then((res) =>
+      console.log(res)
+    );
+  };
+
+  // handler that calls searchCourses from our api-client package
+  const testSearchCourses = (
+    searchQuery: string,
+    minIndex: number,
+    maxIndex: number
+  ) => {
+    SearchAPI.searchCourses(searchQuery, minIndex, maxIndex).then((res) =>
+      console.log(res)
+    );
+  };
+
   return (
     <>
       <h1>GraduateNU Landing Page:</h1>
       <h2>Seasons typed using types from our common package!</h2>
       {seasons.map((season) => (
         <p key={season}>{season}</p>
       ))}
+      <br />
+      <div>
+        <h2>Testing api calls on backend v2</h2>
+        <p>All responses will be logged to the console</p>
+
+        <div>
+          <h3>Auth Routes</h3>
+          <button
+            onClick={async () => {
+              const student = await API.auth.register({
+                fullName: "Aryan Shah",
+                email: "aryan1@gmail.com",
+                password: "aryan1234",
+                academicYear: 2019,
+                graduateYear: 2023,
+                catalogYear: 2019,
+                major: "Computer Science",
+                nuid: "000000000",
+              });
+              console.log(student);
+              setToken(student.accessToken!);
```

Comment:
i dont remember exactly how to implement auth,  but it shouldn't be part of the response body. i think it's either in the set-cookie header, or something else. cc @aryanshah701

Reply:
since currently the backend is setup with access token authentication and not cookies, when logging in the token is returned in the body. 

my backend skills aren't great lmaoo but from all the research i did access token auth flow normally returns the token on signup in the response body, the client has full autonomy on how it wants to store it, and then the client can send it in the Authorization header as a bearer token on each request.

Reply:
i actually wasn't sure, so i did some research. but it seems like the way to go is via cookies 
- https://dev.to/gkoniaris/how-to-securely-store-jwt-tokens-51cf
- (the sample code != what they are saying, but the article seems to say cookies) https://javascript.plainenglish.io/the-ultimate-way-to-store-authentication-tokens-in-javascript-41900756c1ae
- https://stackoverflow.com/questions/51332747/where-to-store-auth-token-frontend-and-how-to-put-it-in-http-headers-of-multip

tldr: there are 3 main ways to store auth token: localstorage, sessionstorage, and cookie. however the first two are bad idea because token can be stolen by CSRF or XSS, so the preferred way is via cookies with some headers to make it secure.

brandon mentioned earlier we can introduce cookies later, though, which sounds good to me